source/features/link-to-gihub-io.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import LinkIcon from 'octicon/link.svg';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+import LinkExternalIcon from 'octicon/link-external.svg';
+
+import features from '.';
+import {getRepositoryInfo} from '../github-helpers';
+
+function getGithubIoLink(): string {
+	return `${getRepositoryInfo().owner!}.github.io`;
+}
+
+async function repoTabInit(): Promise<void> {
+	const repos = select.all<HTMLAnchorElement>('a[href$=".github.io"][itemprop="name codeRepository"]');
+	for (const repo of repos) {
+		const path = new URL(repo.href).pathname.slice(1);
+		const repoInfo = getRepositoryInfo(path);
+
+		repo.parentElement!.append(
+			<a
+				href={`https://${repoInfo.name!}`}
```

Comment:
No need any parsing, the URL is already there:

```suggestion
				href={`https://${repo.textContent!.trim()}`}
```